main.tex

```diff
@@ -521,11 +560,118 @@ \subsection{Over-spending prevention by balance proof}\label{balance}
+	CoinShuffle~\cite{ruffing2014coinshuffle} &$\mathcal{O}(n^2)$ & fixed & decentralized & n.a. & \cmark\footnote{Requires only 2 honest participants.} \\
+	CoinShuffle++~\cite{ruffing2017p2p} & $\mathcal{O}(n)$ & fixed & decentralized & n.a. & \cmark\footnotemark[\value{footnote}] \\
+	CashFusion & $\mathcal{O}(n)$ & variable & centralized & \cmark\footnote{Assuming server does not participate as a verifier.} & \cmark\footnote{Transaction ambiguity guarantees are probablistic, with additional computational difficulty under assumptions.} \\
+	JoinMarket & $\mathcal{O}(n)$ & variable & by taker\footnote{Transactions are initiated and coordinated by users buying mixing offers on a decentralized markeplace.} & n.a.\footnotemark[\value{footnote}] & \cmark \\
+	Chaumian CoinJoin & $\mathcal{O}(n)$ & fixed & centralized & \cmark & \cmark \\
+	This Work & $\mathcal{O}(n)$ & variable & centralized & \cmark & \cmark \\
+\end{tabular}
+\end{minipage}
+\caption{CoinJoin based protocols, no theft risk and requiring a single overt transaction.}
+\label{table:coinjoin}
+\end{table}
+
+\begin{table}[h!]
+\begin{minipage}{\textwidth}
+\renewcommand{\thefootnote}{\thempfootnote} % for multiple references to minipage footnotes
+\centering
+\begin{tabular}{lcccccccc}
+	&&&&&& \multicolumn{3}{c}{Privacy against} \\
+	%\cline{7-9}
+	&Msg. &Txn. & Denom. & Theft & Coord. & Srv. & Part. & Blockchain \\
+
+	Custodial & - & - & - & \xmark & cent. & TTP & TTP & - \\
+	Mixcoin~\cite{bonneau2014mixcoin} & $2n$\footnote{Per mixnet stage}      & $2n$\footnotemark[\value{footnote}] & var.\footnote{Fixed chunk size recommended for privacy, but technically not required.} & TTP\footnote{Provides server accountability} & cent. & TTP & TTP & disjoint \\
+	Blindcoin~\cite{valenta2015blindcoin} & $4n$      & $2n$ & fix.  & TTP\footnotemark[\value{footnote}] & cent. & \cmark & \cmark & disjoint \\
+	TumbleBit~\cite{heilman2017tumblebit} & $12n$      & $4n$ & fix. & \cmark &  cent. & \cmark & \cmark & overt\footnote{TumbleBit provides $k$-anonymity per multi-transaction epoch.} \\
+	CoinSwap\cite{maxwell2013coinswap} & $\mathcal{O}(1)$\footnote{$n=2$} & 4\footnotemark[\value{footnote}] & var. & \cmark & p2p & n.a. & \xmark & disjoint \\
+	Xim~\cite{bissias2014sybil} & n.a. & $7$\footnotemark[\value{footnote}] & var. & \cmark &  p2p\footnote{Advertisements are made on the blockchain} & n.a. & \xmark & disjoint \\
+	CoinParty~\cite{ziegeldorf2015coinparty}& ?$\mathcal{O}(n \log n)$? & 2n & fix. & \cmark\footnote{Assuming $\frac{1}{3}$ of participants are honest.} & dec. & n.a. & \cmark\footnote{Assuming at least 2 participants are honest.} & covert \\
+	CoinSwap\cite{belcher2020design} & $\mathcal{O}(n)$ & $\mathcal{O}(n)$ & var. & \cmark &  taker & n.a. & \cmark & covert \\
+	% A2L?
+\end{tabular}
+\end{minipage}
+\caption{Non-CoinJoin based protocols.}
+\label{table:noncoinjoin}
+\end{table}
```

Comment:
Same, please screenshot.

Reply:
![image](https://user-images.githubusercontent.com/14242/86631612-e2062880-bfbd-11ea-948d-e8db41c5c42a.png)

Reply:
note that CoinParty was omitted from the submitted version, and i still need to go over and verify these one, i suspect there are still some inaccuracies

Reply:
thinking about this table a bit more:

txn should be split into confirmations and something that scales with the number of coins, the number of txns doesn't matter per se for the footprint, and in some cases having `n` as a term is misleading (e.g. coinparty requires 2 confirmation phases, not `n`, and the footprint is on the same order as that of an equivalent coinjoin transaction)

Reply:
@seresistvanandras thoughts?

Reply:
I can't really verify this table is correct and I think ti'd be somewhat counterproductive spending my time on this if the alternative is to progress forward WabiSabi, so I'll ACK it based on I trust your judgements.